<launch>
    <arg name="port" default="/dev/ttyTHS1"/>
    <arg name="baud" default="115200"/>
    <arg name="rate" default="51"/>
    <arg name="name" default="/drivers/remote/signals"/>

    <node name="serial_com_node" pkg="serial_com" type="serial_com" output="screen">
        <param name="port" value="$(arg port)"/>
        <param name="baud" value="$(arg baud)"/>
        <param name="node_rate" value="$(arg rate)"/>
        <param name="serial_info_topic_name" value="$(arg name)"/>
        
        <!-- <rosparam command="load" file="$(find serial_com)/config/serial_com.yaml" />  -->
        <!--Load parameters from config files-->
    </node>
</launch>
